libs/application/templates/parental-leave/src/lib/directorateOfLabour.utils.spec.ts

```diff
@@ -41,6 +41,13 @@ describe('monthsToDays', () => {
 
     expect(daysToMonths(sum)).toBe(7.5)
   })
+
+  it('should return -1.5 months for the maximum period given to the other parent', () => {
```

Comment:
This test is probably not necessary anymore

Reply:
Agree, should be out now.